lib/util/eyeglass_modules.js

```diff
@@ -381,25 +381,37 @@ function dedupeModules(modules) {
   * @param   {Object} pkg - the package.json reference
   * @returns {String} the name of the module
   */
-function getModuleName(pkg) {
+function getModuleName(pkg, pkgPath) {
```

Comment:
`getModuleName` shouldn't need `pkgPath` to complete its job (see comment below)

if it does, you can also reference it off of `pkg.path`